ImageSharp/Program.cs

```diff
@@ -1,38 +1,29 @@
 ﻿using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Formats.Png;
 using SixLabors.ImageSharp.PixelFormats;
 using SixLabors.ImageSharp.Processing;
 
-// Create an empty image canvas that is 256 x 256 px by ImageSharp.
-using var canvas = new Image<Rgba32>(256, 256);
+// Create a canvas from the table image.
+using var canvas = await LoadImage("./assets/table.png");
```

Comment:
By using the table image as the canvas you can automatically preserve any PNG metadata loaded from the image. In this case the table image is an 8bit PNG which means we will reencode it as such, resulting in a much smaller encoded result.

Reply:
@JimBobSquarePants Yes, you are right in this scenario. However, I'm considering comparing these sample codes with another algorithm that uses the `<canvas>` HTML element and its API on a Blazor WebAssembly app. Therefore, I could not reuse the "table" image as a canvas. I need to have an empty image as a canvas and draw the "table"  and the "apple" onto that canvas image.

Reply:
@JimBobSquarePants See also: https://github.com/sample-by-jsakamoto/Blazor-Blending-Two-Images/blob/main/HtmlCanvas/App.razor